spec/functions/authz_core_config_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'apache::authz_core_config' do
+  let(:input1) do
+    {
+      'Require' => [
+        'user foo',
+        'user bar',
+      ]
+    }
+  end
+
+  let(:input2) do
+    {
+      'require_all' => {
+        'require_any' => {
+          'require' => ['user superadmin'],
+          'require_all' => {
+            'require' => ['group admins', 'ldap-group "cn=Administrators,o=Airius"']
+          }
+        },
+        'require_none' => {
+          'require' => ['group temps', 'ldap-group "cn=Temporary Employees,o=Airius"']
+        }
+      }
+    }
+  end
+  # rubocop:disable Style/StringLiterals
+  let(:output2) do
+    [
+      '  <RequireAll>',
+      '    <RequireAny>',
+      '      Require user superadmin',
+      '      <RequireAll>',
+      '        Require group admins',
+      "        Require ldap-group \"cn=Administrators,o=Airius\"",
```

Comment:
Style: we can use single quotes like the other lines to make this more readable:
```suggestion
      '        Require ldap-group "cn=Administrators,o=Airius"',
```